WebRoot/templates/calendar/Appointment.template

```diff
@@ -1,219 +1,329 @@
-                    <td><div id='${id}_reminderConfigure'></div></td>
-                </tr>
-
-                <tr id="${id}_meetingInPastWarning" style="display:none">
-                    <td></td>
-                    <td>
-                        <div class="ImgInformation" style="float: left; margin: 0 0.5rem 0 0;"></div>
-                        <div id="${id}_meetingInPastWarningMessageText" style="font-size: 0.95rem;"></div>
-                    </td>
-                </tr>
-
-                <tr>
-                    <td class='ZmFieldLabelRight'><$= data.isAppt ? ZmMsg.displayLabel : ZmMsg.taskFolder $></td>
-                    <td><table role="presentation" class='ZPropertySheet' cellspacing='6'>
-                            <tr><td id='${id}_showAsSelect' align="right" style='padding-left:0;'></td>
-                                <td id='${id}_folderSelect'></td>
-                                <td>
-                                    <input type='checkbox' id='${id}_privateCheckbox'>
-                                    <label for="${id}_privateCheckbox" style='margin-left:.25em;white-space:nowrap;'><$= ZmMsg._private $></label>
-                                </td>
-                            </tr>
-                        </table>
-                    </td>
-                    <td></td>
-                    <td colspan='2'><div id='${id}_reminderEmailCheckbox'></div></td>
-                </tr>
-                <$ if (appCtxt.get(ZmSetting.CAL_DEVICE_EMAIL_REMINDERS_ENABLED)) { $>
-                    <tr><td colspan='2'></td>
-                        <td></td>
-                        <td colspan='2'><div id='${id}_reminderDeviceEmailCheckbox'></div></td>
-                    </tr>
-                <$ } $>
-
-                <tr id='${id}_attachment_container' style="display:none;">
-                </tr>
-
-                <tr>
-                    <td colspan='5' id='${id}_scheduler_option' style='padding:7px;'>
-                        <div style="float:left;" class="calendar_edit_field"><$= ZmMsg.scheduler $></div>
-                        <div style="float:left; padding-left:5px;" id="${id}_scheduleButton" class="FakeAnchor"><$=
-                        ZmMsg.show $></div>
-                        <div id="${id}_scheduleImage" class="ImgSelectPullDownArrow" style="float:left;"></div>
-                    </td>
-                </tr>
-
-                <tr>
-                    <td colspan='5' id='${id}_scheduler'></td>
-                </tr>
-
-                </table>
-
-
-            </td>
-            <td width="8"></td>
-            </tr>
-
-        </table>
-    </div>
-
-    <!-- appointment notes -->
-    <div id='${id}_notes' style='padding:5px;'></div>
+	<div id='${id}_top'>
+		<table role="presentation" width='100%' id='${id}_table'>
+			<tbody>
+				<tr>
+					<td colspan="2">
+						<div class="ZmApptEditHeader">
+							<h3>Appointment</h3>
```

Comment:
Should this be translated?

Reply:
Fixed: https://github.com/Zimbra/zm-web-client/pull/143/commits/b15e5e1cd57ec3595682a88031056b2ef693148d#diff-60a45c724f7aa5b44add9c6b951de7d2R19

Thanks for pointing this out.